src/Apps/Legion/Services/Utils.ts

```diff
@@ -0,0 +1,29 @@
+import { IConsoleLogFunction } from '../Models/Types';
+
+export function getDebugLogger(
```

Comment:
So I asked this to Matt and he said we can use the existing methods or whatnot by referencing, and he said dont worry about dependencies (so no duplicates). Can you confirm @darsney?

Reply:
removed duplicates